src/forklift/core.py

```diff
@@ -78,6 +78,8 @@ def remove_temp_table(table):
         if not arcpy.Exists(crate.destination):
             log.debug('%s does not exist. creating', crate.destination)
             _create_destination_data(crate)
+            if arcpy.Exists(path.join(hash_gdb_path, crate.name)):
```

Comment:
in what scenario does the crate destination not exist but the table would already exist in the hash table?